cmd/promoted-image-governor/main.go

```diff
@@ -80,6 +99,18 @@ func (o *options) validate() error {
 	return nil
 }
 
+func load(path string) (*OpenshiftMappingConfig, error) {
```

Comment:
nit: Since this exclusively loads an OpenshiftMappingConfig, maybe it would be better to call the function `loadMappingConfig` or something.

Reply:
Updated.